baseapp-auth/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = baseapp_auth
-version = 0.4.6
+version = 0.5.0
```

Comment:
Isn't that a new major release? All funcitions/classes from the previous version won't work anymore.

Reply:
Yes, this is a major change to baseapp and requires a major version release